images/pulp/stable/Containerfile

```diff
@@ -39,3 +41,10 @@ USER root:root
 RUN ln $(pip3 show pulp_ansible | sed -n -e 's/Location: //p')/pulp_ansible/app/webserver_snippets/nginx.conf /etc/nginx/pulp/pulp_ansible.conf
 RUN ln $(pip3 show pulp_container | sed -n -e 's/Location: //p')/pulp_container/app/webserver_snippets/nginx.conf /etc/nginx/pulp/pulp_container.conf
 RUN ln $(pip3 show pulp_python | sed -n -e 's/Location: //p')/pulp_python/app/webserver_snippets/nginx.conf /etc/nginx/pulp/pulp_python.conf
+
+RUN <<EOF
+  if [[ "$PULP_UI_URL" ]]; then
+    mkdir -p "${PULP_STATIC_ROOT}pulp_ui"
+    curl -Ls $PULP_UI_URL | tar -xzv -C "${PULP_STATIC_ROOT}pulp_ui"
+  fi
+EOF
```

Comment:
I haven't seen this "here document" thing in a container file before.
Can you try with `\` on each lines end instead?

Reply:
I couldn't get it to work with `\`.

Reply:
Ok I got a version to work with `\` to work, had to switch to using single `[` for the if check.